db/DatabaseManager.py

```diff
@@ -0,0 +1,90 @@
+import sqlite3
+
+"""This class initiates a connection to the sqlite database
+and returns a cursor object upon quering
+"""
+
+
+class DatabaseManager(object):
+
+    """Connects to database and returns a cursor object
+    """
+
+    def __init__(self, db):
+        self.conn = sqlite3.connect(db)  # connects to Amity.sqlite database
+        self.conn.commit()
+        self.cur = self.conn.cursor()
+        # checks if atleast 1 table exists (sqlite_sequence)
+        if self.gettablecount() < 2:
+            self.createtables()  # if so, create the remaining 3
+
+    """Performs a query to the database based on the supplied
+    query string.This method then returns the cursor object to the
+    calling class/method
+    """
+
+    def query(self, arg):
+        self.cur.execute(arg)
+        self.conn.commit()
+        return self.cur
+
+    """Called when the class object is destroyed.
+    """
+
+    def __del__(self):
+        # Closes the connection to the database to prevent database locks
+        self.conn.close()
+
+    """Returns list of predefined rooms
+    """
+
+    def getrooms(self):
+        list_offices = [['Valhalla', 6, 'OFFICE'],
```

Comment:
**Code Quality: Nitpick**

This is a good place to have used of a dictionary instead of using nested lists.